configs/components/puppet-ca-bundle.json

```diff
@@ -1,4 +1,4 @@
 {
-  "url": "git://github.com/puppetlabs/puppet-ca-bundle.git",
+  "url": "git@github.com:puppetlabs/puppet-ca-bundle.git",
```

Comment:
I don't know what the best answer is... I guess cherry-picking them to stable would be the cleanest thing to do?